src/ripple/app/main/GRPCServer.cpp

```diff
@@ -438,7 +438,7 @@ GRPCServerImpl::start()
 }
 
 void
-GRPCServer::run()
+GRPCServer::onStart()
 {
     // Start the server and setup listeners
     if ((running_ = impl_.start()))
```

Comment:
I get what this line is doing, but using an extra set of parentheses to prevent the compiler warning is pretty subtle.  I think this line can be made easier to read if we use C++17's if statement with initializer.  So I'll suggest changing the line to look like this:
```
if (running_ = impl_.start(); running_)
```

Reply:
Someone on the team was recently muttering about how we don't give threads names.  Well, we're just about to create a thread; let's give it a name.  One way to do that would be to change the code inside this `if` to look like this:
```
        thread_ = std::thread([this]() {
            // Start the event loop and begin handling requests
            beast::setCurrentThreadName("rippled: grpc");
            this->impl_.handleRpcs();
        });
```
If you choose to make this change you'll need to add a
```
#include <ripple/beast/core/CurrentThreadName.h>
```
at the top of the file.